versions.tsv

```diff
@@ -334,7 +334,6 @@ pixman	0.34.0_1
 pkg-config	0.29.2_1
 plink	1.07
 poa	2.0
-poretools	0.6.0_6
```

Comment:
This file `versions.tsv` is auto-generated. Please remove this line. When we're ready to release a new version of ORCA, I'll show you how to update this file. Thanks for being so thorough!